frontend/src/metabase/pulse/components/PulseEdit.jsx

```diff
@@ -148,6 +158,27 @@ export default class PulseEdit extends Component {
           </ModalWithTrigger>
         </div>
         <div className="PulseEdit-content pt2 pb4">
+          <Flex
+            bg={color("bg-medium")}
+            p={2}
+            my={2}
+            align="top"
+            style={{ borderRadius: 8 }}
+            className="hover-parent hover--visibility"
+          >
+            <Icon name="warning" color={color("warning")} size={24} mr={1} />
+            <Box ml={1}>
+              <h3>{t`Pulses are being phased out`}</h3>
+              <p className="m0 mt1 text-medium text-bold">{jt`You can now set up ${link} instead. We'll remove Pulses in a future release, and help you migrate any that you still have.`}</p>
+            </Box>
+            <Icon
+              className="hover-child text-brand-hover cursor-pointer bg-medium"
+              name="close"
+              ml="auto"
+              onClick={() => this.dismissPinMessage()}
```

Comment:
Doesn't look like this handler actually does anything at the moment.

Reply:
Whoops, copypasta.